src/components/basecamp/Organizations.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useEffect, useState } from 'react'
+import { useQuery } from '@apollo/client'
+import { formatDistanceToNow } from 'date-fns'
+import { useSession, signIn } from 'next-auth/react'
+import { OrganizationContentType, OrganizationType } from '../../js/types'
+import { orgsToCsv, saveAsCSVFile } from '../../js/utils/csv'
+import CreateUpdateModal from './CreateUpdateModal'
+import OrganizationForm from './OrganizationForm'
+import { graphqlClient } from '../../js/graphql/Client'
+import { QUERY_ORGANIZATIONS } from '../../js/graphql/gql/organization'
+import { toast } from 'react-toastify'
+
+export default function Organizations (): JSX.Element {
+  const session = useSession()
+  useEffect(() => {
+    if (session.status === 'loading') {
+      return
+    }
+    if (session.status !== 'authenticated') {
+      void signIn('auth0', { callbackUrl: '/basecamp/organizations' })
+    }
+  }, [session])
+
+  const isAuthorized = session.status === 'authenticated' && session?.data?.user.metadata?.roles?.includes('user_admin')
+
+  return (
+    <>
+      {!isAuthorized && <div>You're not authorized to view this page.</div>}
+      {isAuthorized && <OrganizationTable />}
+    </>
+  )
+}
+
+const OrganizationTable = (): JSX.Element => {
+  const [modalOpen, setModalOpen] = useState(false)
+  const [focussedOrg, setfocussedOrg] = useState<OrganizationType | null>(null)
+
+  const { data, error } = useQuery(
```

Comment:
The User page uses `useSWR` but I don't think it works here because that's calling the Auth0 API, while here our Apollo GraphQL client already has caching and refetching logic. 

I wasn't sure whether to fetch data server-side. But I figure client-side is fine given that users are admins, so performance is less important. Also data access is more adhoc so less opportunities to benefit from page-level caching. 

https://nextjs.org/docs/basic-features/data-fetching/client-side

Reply:
For admin dashboard client side fetching is perfectly fine :)